projects/plugins/jetpack/extensions/plugins/publicize/settings.js

```diff
@@ -1,60 +1,25 @@
 import {
-	PostPublishManualSharing,
-	PostPublishReviewPrompt,
 	PublicizePanel,
 	SocialImageGeneratorPanel,
-	useSyncPostDataToStore,
 	usePublicizeConfig,
-	useSocialMediaConnections,
 } from '@automattic/jetpack-publicize-components';
-import { JetpackEditorPanelLogo } from '@automattic/jetpack-shared-extension-utils';
-import { PluginPrePublishPanel } from '@wordpress/edit-post';
-import { PostTypeSupportCheck } from '@wordpress/editor';
-import { __ } from '@wordpress/i18n';
-import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
 import UpsellNotice from './components/upsell';
+import PostPublishPanels from './post-publish';
+import PrePublishPanels from './pre-publish';
 
-export const Settings = () => {
-	useSyncPostDataToStore();
-	const { hasEnabledConnections } = useSocialMediaConnections();
+const Settings = () => {
 	const { isSocialImageGeneratorAvailable } = usePublicizeConfig();
 
 	return (
-		<PostTypeSupportCheck supportKeys="publicize">
-			<JetpackPluginSidebar>
-				<PublicizePanel>
-					<UpsellNotice />
-				</PublicizePanel>
-				{ isSocialImageGeneratorAvailable && <SocialImageGeneratorPanel /> }
-			</JetpackPluginSidebar>
-
-			<PluginPrePublishPanel
-				initialOpen={ hasEnabledConnections }
-				id="publicize-title"
-				title={
-					<span id="publicize-defaults" key="publicize-title-span">
-						{ __( 'Share this post', 'jetpack' ) }
-					</span>
-				}
-				icon={ <JetpackEditorPanelLogo /> }
-			>
-				<PublicizePanel prePublish={ true }>
-					<UpsellNotice />
-				</PublicizePanel>
-			</PluginPrePublishPanel>
-
-			{ isSocialImageGeneratorAvailable && (
-				<PluginPrePublishPanel
-					initialOpen
-					title={ __( 'Social Image Generator', 'jetpack' ) }
-					icon={ <JetpackEditorPanelLogo /> }
-				>
-					<SocialImageGeneratorPanel prePublish={ true } />
-				</PluginPrePublishPanel>
-			) }
-
-			<PostPublishManualSharing />
-			<PostPublishReviewPrompt />
-		</PostTypeSupportCheck>
+		<>
+			<PublicizePanel>
```

Comment:
Something about this change means that without the sidebar open, none of the Publicize settings are rendered in the pre-publish panel. I think it might be that the Panel is no longer contained in the JetpackPluginSidebar component or something.

@monsieur-z Any ideas?


https://github.com/Automattic/jetpack/assets/96462/b64449ad-a392-4c43-9c6f-d7f9565607ab

Reply:
Sorry @pablinos, I was AFK. It seems this was taken care of. Let me know if you still need my help.